ConsumerApi/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -100,7 +100,7 @@ public static IServiceCollection AddCustomAspNetCore(this IServiceCollection ser
 
         services.AddTransient<IUserContext, AspNetCoreUserContext>();
 
-        if (configuration.Modules.Devices.Infrastructure.PushNotifications.Providers.Sse is { Enabled: true })
+        if (configuration.Modules.Devices.Infrastructure.PushNotifications.Providers.Sse is { Enabled: true, EnableHealthCheck: true })
```

Comment:
It shouldn't be possible to disable the SSE Server health check

Reply:
done